### PR TITLE
windows 10 tutorial

### PR DESCRIPTION
**Pull request by HenryKobin**
Hey guys! I got jekyll running flawlessly on my copy of windows 10 creators update with some pretty basic steps taken. I was curious if I could contribute this guide to the docs. Thanks for your time, let me know if you want me to edit anything / reformat the markdown.

/cc @parkr  @jekyllbot

### CODE REVIEW

**Comment by tomjohnson1492** [comment]
Henry, thanks for writing out this installation doc. I think it would fit better on the [Jekyll on Windows](http://jekyllrb.com/docs/windows/#installation) page rather than in Tutorials. 

Also, how do the rest of the Windows instructions fit in with your Windows 10 section? Do you still have to complete the other Windows-related tasks such as Time Zone Management or Auto-regeneration? Does the Chocolately tutorial still work on Win 10? In short, can you integrate this info more seamlessly into the larger Windows documentation (on that page)?

**Comment by HenryKobin** [comment]
Thanks for the response @tomjohnson1492. I agree it would probably fit better on that page. Auto-Regeneration is working fine, although the error message still displays as a warning. I'll have to double check time-zone management, but my files were formatting with the current time/date when I create a new project. 
I am not sure about the Chocolately method, I used Bash so I never had to try. I can certainly double check this. Let me get back to you guys in the next 24 hours with a version of the Jekyll on Windows page. 
Should I create a new pull request when I finish or just continue this one?

**Comment by DirtyF** [comment]
@HenryKobin There is no need to create another PR, just continue here. 

/cc @jekyll/windows for feedback

File: docs/_docs/windows.md [hunk 1]
```diff
@@ -8,7 +8,55 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
...
+
+Now we can install Ruby(2.3). First we need to add the repository list from BrightBox.
+
+```
+sudo apt-add-repository ppa:brightbox/ruby-ng
+sudo apt-get update
+sudo apt-get install ruby2.3 ruby2.3-dev build-essentials
+```
+
+Or you can install an old version of ruby (1.9.3).
```
Comment (DirtyF) [inline docs/_docs/windows.md]: I wouldn't mention this, Jekyll requires Ruby > 2.0

File: docs/_docs/windows.md [hunk 2]
```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
...
+```
+
+You can test by running:
+
+```
+jekyll new my_project
+```
+
+
+**And thats it!**
```
Comment (pup) [inline docs/_docs/windows.md]: `that's`

File: docs/_docs/windows.md [hunk 3]
```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
...
+If you cd into the folder, you can make sure time management is working by opening your "_posts"_ folder. You should see a markdown file with the current date listed.
```
Comment (pup) [inline docs/_docs/windows.md]: cd should be `cd` imo

Comment (pup) [inline docs/_docs/windows.md]: `"_posts"_` is probably the wrong markup for this? consider using backticks here

File: docs/_docs/windows.md [hunk 4]
```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
...
+
+*Please note* Bash on Ubuntu on Windows is still under development, so you may run into issues. If you see an Auto-Regeneration error warning in your Bash instance, you can ignore it.
+
+## Installation via Chocolatey
+
 
 A quick way to install Jekyll is to follow the [installation instructions by David Burela](https://davidburela.wordpress.com/2015/11/28/easily-install-jekyll-on-windows-with-3-command-prompt-entries-and-chocolatey/):
```
Comment (pup) [inline docs/_docs/windows.md]: maybe amend this to say `A quick way to install Jekyll using Chocolatey is [...]`?

File: docs/_docs/windows.md [hunk 5]
```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
...
+```
+
+Now we can install Ruby. First we need to add the repository list from BrightBox.
+
+```
+sudo apt-add-repository ppa:brightbox/ruby-ng
+sudo apt-get update
+sudo apt-get install ruby2.3 ruby2.3-dev build-essentials
+```
+Next lets update our Ruby gems:
```
Comment (pup) [inline docs/_docs/windows.md]: `let's`

File: docs/_docs/windows.md [hunk 6]
```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
...
+```
+bash
+```
+Your Command Prompt instance should now be a Bash instance. Now we must update our repo lists and packages.
+
+```
+sudo apt-get update -y && sudo apt-get upgrade -y
+```
+
+Now we can install Ruby. First we need to add the repository list from BrightBox.
```
Comment (pup) [inline docs/_docs/windows.md]: could you provide a link here? i could see folks getting sceptical at just adding a PPA with no real context attached to it

File: docs/_docs/windows.md [hunk 7]
```diff
@@ -8,7 +8,56 @@ Jekyll with the proper tweaks. If you are using Windows 10 Anniversary Update,
 the easiest way to run Jekyll is to use the new [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about?f=255&MSPPError=-2147217396).
 For older installations, this page aims to collect some of the general knowledge and lessons that have been unearthed by Windows users.
 
-## Installation
+## Installation via Bash on Windows 10
+
+*Please note:* You must have [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about?f=255&MSPPError=-2147217396) enabled.
+
+First lets make sure all our packages / repositories are up to date. Open a new Command Prompt instance, and type the following:
```
Comment (pup) [inline docs/_docs/windows.md]: `let's`

**Comment by ashmaroli** [comment]
@HenryKobin can you also test if TimeZone Management is fine during the build process on the subsystem..? I mean, say the `_config.yml` contains `timezone: America/New_York` Does the date and time change accordingly on your `HTML` files in the `_site` folder?
Thanks! :smiley:

**Comment by HenryKobin** [comment]
@ashmaroli Timezone management from the config file works just fine. I tested it with America/New_York and it showed up properly via liquid templating.
@pup I corrected those errors, and removed some subheaders from the chocolatey tutorial so that it would be clear that those portions don't apply to the Bash on Ubuntu on Windows situation. I also tested Timezone via the config file, and it showed up correctly in the HTML. Also, auto-regeneration is working.
cc/ @jekyll/windows

**Comment by DirtyF** [comment]
Thanks @HenryKobin great addition for Windows users 🐚  

@jekyllbot : merge +docs

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Pull request merged**